src/ansi-c/library/cprover_contracts.c

```diff
@@ -1286,6 +1286,10 @@ void *__CPROVER_contracts_write_set_havoc_get_assignable_target(
   __CPROVER_size_t idx)
 {
 __CPROVER_HIDE:;
+#ifdef DFCC_DEBUG
```

Comment:
How to set this/when is this set?

Reply:
these debug macros should be set at build time but there's no direct way to do that yet.